### DISCUSSION TITLE
DataHubBundle - DataHub Config not available in settings

### DISCUSSION THREAD

**Discussion by xalessandrox** [pimcore/pimcore#14108] [Q&A / Support] [answered]
Hello everyone,
I am new to Pimcore. My version is 10.5.15. I needed to add DataHub so I followed instructions:
I executed composer command, then enabled and installed the bundle.
The same was done for DataHubImporter.
The DataHub icon now appears on the side navigation and I can add graphQL and ObjectsImporter endpoint.
When I click on Settings, though, the DataHubConfig option is not available.
Here is what I mean: 

![image](https://user-images.githubusercontent.com/74103815/213479469-0f954d24-1d30-4526-8953-52ec5ded719d.png)

 
I thought in the settings' list I would find at the end DataHubConfig, like in the course that i am following.
I reinstalled the bundles a couple of times but nothing changed. Am I missing something?
Here is the output from CLI:
![image](https://user-images.githubusercontent.com/74103815/213494405-12b99155-85bb-4552-b452-bda977376dd2.png)

Any help would be appreciated. thanks!

**Comment by kingjia90** [comment, marked as answer]
You are already in the DataHub Config in your screenshot.
![image](https://user-images.githubusercontent.com/6014195/213538240-fe01b6cf-91ec-48a4-a7dc-484bb019673c.png)

The screenshot in https://pimcore.com/docs/data-hub/current/Basic_Principle.html is outdated.
The icon was moved to the top level, instead of settings' submenu

**Reply by xalessandrox** [reply]
This was an easy one ;) Thank you for the answer! Now I know everything works fine.
